File: KH.Dto/common/CommonLookupFiltersRequest.cs
namespace KH.Dto.common
{
  public class CommonLookupFiltersRequest : PagingRequestHelper
  {
    public long? Id { get; set; }
    //public string? Search { get; set; } //for multiple use NameEn - NameAr - Description
    public string? NameEn { get; set; }
    public string? NameAr { get; set; }
    public string? Description { get; set; }
    public bool? IsDeleted { get; set; } = false;
    public bool? RequestFromCustomerPortal { get; set; } = false;


  }
}

File: KH.WebDashboard/src/app/@core/models/base/request/common-lookup-filters-request.ts

import { CustomPagingParameters } from "../custom-paging-parameters";

export interface CommonLookupFiltersRequest extends CustomPagingParameters  {
  id?: number;
  nameEn?: string;
  nameAr?: string;
  description?: string;
  model?: string;
  modelId?: number;
  isDeleted?: boolean;
  IgnoreCache?: boolean | null ;
  isSent?: boolean;
  requestFromCustomerPortal?: boolean;

  
}
